src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -288,13 +237,16 @@ private String getAppointeeDescription(SubscriptionType subscriptionType, SscsCa
         }
     }
 
-    private void subscriptionDetails(Map<String, String> personalisation, Subscription subscription, Benefit benefit) {
+    private void subscriptionDetails(Map<String, String> personalisation, Subscription subscription, Benefit benefit, SscsCaseData sscsCaseData) {
         final String tya = tya(subscription);
         personalisation.put(APPEAL_ID, tya);
         if (benefit != null) {
             personalisation.put(MANAGE_EMAILS_LINK_LITERAL, config.getManageEmailsLink().replace(MAC_LITERAL, getMacToken(tya, benefit.name())));
         }
         personalisation.put(TRACK_APPEAL_LINK_LITERAL, config.getTrackAppealLink() != null ? config.getTrackAppealLink().replace(APPEAL_ID_LITERAL, tya) : null);
```

Comment:
Might be clearer if you moved this line to be the 'else' statement of your new if statement below.

Reply:
Change implemented